src/components/DatePicker/style.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/css';
+
+const small = '0.25rem 0.5rem';
+const medium = '0.5rem 1rem';
+const large = '1.5rem 2rem';
```

Comment:
could be more specific with the name, small(What)? people will need to read your entire style file to know what size is this variable for.